README.md

```diff
@@ -1,2 +1,91 @@
+
+  # Copy the UUID4 from Uptycs' UI
+  # Uptycs' UI : "Cloud"->"AWS"->"Integrations"->"ACCOUNT INTEGRATION"
+  # You can generate your own UUID. If you do, make sure Uptycs' UI is updated with it
+  external_id = "465308b9-fadb-449b-8d2d-3b5b3f2457f9"
+
+  # CloudTrail source: S3 Bucket or Kinesis stream?
+  # Set either `cloudtrail_s3_bucket_name` or `kinesis_stream_name` to allow Uptycs to ingest CloudTrail events
+  # Provide the S3 bucket name which contains the CloudTrail data
+  cloudtrail_s3_bucket_name = ""
+
+  # Name of the Kinesis stream configured to stream CloudTrail data
+  kinesis_stream_name = ""
+
+  # Name of the S3 bucket that contains the VPC flow logs
+  vpc_flowlogs_bucket_name = ""
+
+  tags = {
+    Service = "cloudquery"
+  }
+}
+
+output "aws_parameters" {
+  value = module.iam-config.aws_parameters
+}
+
+```
+
+## 2. Set Profile and Region before execute terraform
+
+```sh
+export AWS_PROFILE="< profile name >"
+export AWS_DEFAULT_REGION="< pass region >"
+```
+
+## 3. Execute Terraform script
+
+```sh
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+## Inputs
+
+
+| Name                      | Description                                                     | Type     | Default             | Required |
+| --------------------------- | ----------------------------------------------------------------- | ---------- | --------------------- | ---------- |
+| integration_name          | Prefix to be used for naming new resources                      | `string` | `UptycsIntegration` |          |
+| upt_account_id            | Uptycs AWS account ID                                           | `string` | `""`                | Yes      |
+| aws_account_id            | AWS organization's master account ID                            | `string` | `""`                | Yes      |
+| external_id               | Role external ID provided by Uptycs                             | `string` | `""`                | Yes      |
+| vpc_flowlogs_bucket_name  | Name of the S3 bucket that contains the VPC flow logs           | `string` | `""`                |          |
+| cloudtrail_s3_bucket_name | Name of the S3 bucket which contains the CloudTrail data        | `string` | `""`                |          |
+| kinesis_stream_name       | Name of the Kinesis stream configured to stream CloudTrail data | `string` | `""`                |          |
+| tags                      | Tags to apply to the resources created by this module           | `map`    | `{}`                |          |
+
+## Outputs
+
+
+| Name           | Description                                     |
+| ---------------- | ------------------------------------------------- |
+| aws_parameters | AWS parameters (ExternalId and IntegrationName) |
+
+## Notes:-
+
+- The user should have `Administrators`  permission to the master account to create resources.
+- Every child account in the organization shoud have `OrganizationAccountAccessRole` role.
```

Comment:
Add this link for more information
https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_access.html